sdk/iothub/src/lib.rs

```diff
@@ -1 +1,4 @@
+#[macro_use]
+extern crate serde_json;
```

Comment:
nit: I'd prefer not to globally import macros, and to just refer to them like `serde_json::some_macro!`

Reply:
Didn't know about this, thanks!